### ISSUE TITLE
Use tags for each course module

### ISSUE THREAD

**Issue by jbenhamou** [eugenp/REST-With-Spring#68]
### Background

I just bought your course and I’m unsure how to checkout the source code that corresponds to a particular module. I did checkout the branch corresponding to the course (e.g. course1).
### Problem

Checking out the course branch leaves me at the tip of the branch, essentially at the end of the course.
It seems there are not tags that I could use to checkout points in the course based on each video.
What do you recommend I should do?
### Proposed Solution

My general expectation was that the students will be able to work through the problems in real time with the videos. This requires a well defined starting point that allows students to grapple with the problem at hand, rather than working with code that historically showed different ways of solving the same problem.

**Comment by eugenp** [comment]
Hey @jbenhamou - sorry for the late response on this one, I somehow marked it as read and didn't get back to it. 
This is definitely a good good question/suggestion. 
Originally, I decided to only include the finished versions of the code. That's mostly because - when I published my Packt course and provided these detailed start and finished points in the code, almost everyone was asking for the finished version.
However, given the fact that I have received a few requests for these checkpoints in the code - I'm reconsidering that decision. Ultimately it's still a tiny tiny section out of all the students going through the course - maybe 2%, but - if it's going to be used, I'm happy to put that together. So - it's on my TODO list for the summer.
Hope that helps. Cheers,
Eugen.
